### PR TITLE
css-pseudo-classes - Henry Pham

### PR DESCRIPTION
**Pull request by hqpham98**
image
image

### CODE REVIEW

File: css-pseudo-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Hover Button</h1>
+    <button class="button">I'm a button!</button>
+  </body>
```
Comment (shawnkost) [inline css-pseudo-classes/index.html:12]: Missing a `table`

File: css-pseudo-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+button {
+  padding: 6px 12px;
+  font-size: 20px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+  cursor: pointer;
+}
```
Comment (shawnkost) [inline css-pseudo-classes/styles.css:10]: Missing all the styles for your `table`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Henry!

**Comment by hqpham98** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**